src/pudl/transform/ferc1.py

```diff
@@ -751,6 +751,35 @@ def assign_quarterly_data_to_yearly_dbf(
     return df
 
 
+class AddColumnWithUniformValue(TransformParams):
+    """Parameters for adding a column to a table with a signle value."""
+
+    column_value: Any = None
+    is_dimension: bool = False
+
+
+class AddColumsWithUniformValue(TransformParams):
```

Comment:
Should be `Columns` not `Colums` I think? As discussed making `Value` into `Values` might also help visually differentiate these

Reply:
lol again good catch!